save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/ManageGitCredentialsCard.kt

```diff
@@ -161,10 +169,10 @@ fun manageGitCredentialsCardComponent() = FC<ManageGitCredentialsCardProps> { pr
         div {
             className = ClassName("row d-flex justify-content-center")
             div {
-                className = ClassName("col-3 d-sm-flex align-items-center justify-content-center")
+                className = ClassName("col-11 text-right")
                 button {
                     type = ButtonType.button
-                    className = ClassName("btn btn-sm btn-primary")
+                    className = ClassName("btn btn-sm btn-outline-primary")
                     onClick = {
                         setGitCredentialToUpsert(GitDto.empty)
                         setUpdateFlag(false)
```

Comment:
I'd suggest using buttonBuilder where possible

Reply:
without a testing, I am afraid to change all buttons, I am afraid, that I will break something...
Will do it later, but now let's have a rule to use only buttonBuilder instead of anything else.